instrumentation/opentelemetry_oban/lib/opentelemetry_oban/plugin_handler.ex

```diff
@@ -41,11 +43,12 @@ defmodule OpentelemetryOban.PluginHandler do
       @tracer_id,
       "#{plugin} process",
       metadata,
-      %{}
+      %{attributes: %{"oban.plugin" => plugin}}
```

Comment:
This should use atoms.
```suggestion
      %{attributes: %{"oban.plugin": plugin}}
```